### PR TITLE
Salty Runback & Quick Exit (Wifi-Compatible)

### LINKED ISSUE #28: Salty Runback + Quick Exit Implementation
LRASTART should be a button combo that checks to be held instead of recursively pressed, this allows you to ergonomically do the combo faster in one motion, and without worrying about order, this is something pm has and it feels really nice

along with that, the yes no prompt should either immediately open, or be skipped altogether
imo please skip it altogether once the button combo held check mentioned above is true

ergonomic extra button held for different actions, as follows:
* LRASTART   - no contest screen, no prompt to confirm Yes/No
* LRASTART+X - quick exit, go back to sss
* LRASTART+Y - salty runback

### PR DESCRIPTION
**Pull request by blu-dev**
Resolves #28 aside from LRASTART without X+Y being exit to No Contest.

### CODE REVIEW

File: fighters/mario/src/opff.rs [hunk 1]
```diff
@@ -213,6 +213,12 @@ pub unsafe fn moveset(fighter: &mut L2CFighterCommon, boma: &mut BattleObjectMod
     noknok_timer(fighter, boma, id);
     noknok_reset(fighter, id, status_kind);
     noknok_training(fighter, id, status_kind);
+
+    if fighter.is_button_on(Buttons::AppealHi) {
```
Comment (techyCoder81) [inline fighters/mario/src/opff.rs]: hey actually why is this here?

**Review by techyCoder81 — CHANGES_REQUESTED** [review]

File: fighters/mario/src/opff.rs (hunk 1 above)
Comment (blu-dev) [reply fighters/mario/src/opff.rs]: Accident left in for when I was testing. Has been resolved.

**Review by techyCoder81 — APPROVED** [review]
